code/game/objects/items/storage/boxes/service_boxes.dm

```diff
@@ -209,9 +209,14 @@
 	desc = "A box full of random stickers. Do give to the clown."
 
 /obj/item/storage/box/stickers/PopulateContents()
-	var/list/types = subtypesof(/obj/item/sticker)
+	var/static/list/noncontraband = list()
+	if(!LAZYLEN(noncontraband))
```

Comment:
Instead of this, move the init logic to a proc, thus letting you do:

```dm
var/static/list/non_contraband = generate_non_contraband_stickers_list()
```

(also `non_contraband` over `noncontraband`)